packages/analytics-js-plugins/src/bugsnag/utils.ts

```diff
@@ -141,17 +142,15 @@ const loadBugsnagSDK = (externalSrcLoader: IExternalSrcLoader, logger?: ILogger)
     return;
   }
 
-  externalSrcLoader
-    .loadJSFile({
-      url: BUGSNAG_CDN_URL,
-      id: ERROR_REPORT_PROVIDER_NAME_BUGSNAG,
-      callback: () => {
-        logger?.debug('Bugsnag script loaded');
-      },
-    })
-    .catch(e => {
```

Comment:
do errors get caught internally in externalSrcLoader loadJSFile method?

Reply:
Yes.